diff --git a/dist-cjs/check/verifyPackageTree.js b/dist-cjs/check/verifyPackageTree.js
index ac007c4c3f0bec1fdfff5cc724b8475dc357735c..21304d79670a3887b090d7b960e07f8b725c02ac 100644
--- a/dist-cjs/check/verifyPackageTree.js
+++ b/dist-cjs/check/verifyPackageTree.js
@@ -22,7 +22,7 @@ function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj &&
 
 const DEPS_TO_CHECK = [
 // These are packages most likely to break in practice.
-'esbuild', 'eslint', 'webpack', 'webpack-dev-server'];
+'esbuild', 'webpack-dev-server'];
 
 // We assume that having wrong versions of these
 // in the tree will likely break your setup.
